site/examples/users.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Users sounds a little weird... because using it in their webpages are users too.